### ISSUE TITLE
Rule "missing-playwright-await" incorrectly reports error on toMatchSnapshot

### ISSUE THREAD

**Issue by ssams** [playwright-community/eslint-plugin-playwright#31]
The following snippet (incorrectly) generates an error from the "missing-playwright-await" rule:

```typescript
const screenshot = await page.locator('#demo').screenshot();
expect(screenshot).toMatchSnapshot('demo.png');
```

The rule wants to enforce an `await` for the expect call, `toMatchSnapshot` however does not return a Promise but is synchronous. References:
* https://playwright.dev/docs/1.16/test-snapshots
* https://playwright.dev/docs/test-assertions#expectvaluetomatchsnapshotname-options
* https://github.com/microsoft/playwright/blob/e37660b068c9f9c6018dad7f999bf4f8de4e6f81/packages/playwright-test/src/matchers/toMatchSnapshot.ts

Version eslint-plugin-playwright: 0.6.0
